tracardi/process_engine/action/v1/connectors/elasticsearch_query/plugin.py

```diff
@@ -0,0 +1,115 @@
+from tracardi_plugin_sdk.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi_plugin_sdk.action_runner import ActionRunner
+from .model.config import Config, Credentials
+from elasticsearch import AsyncElasticsearch, ElasticsearchException
+from tracardi.service.storage.driver import storage
+from tracardi.domain.resource import ResourceCredentials
+from tracardi_plugin_sdk.domain.result import Result
+
+
+def validate(config: dict):
+    return Config(**config)
+
+
+class ElasticSearchFetcher(ActionRunner):
+
+    @staticmethod
+    async def build(**kwargs) -> 'ElasticSearchFetcher':
+        config = validate(kwargs)
+        credentials = await storage.driver.resource.load(config.source.id)
+        return ElasticSearchFetcher(config, credentials.credentials)
+
+    def __init__(self, config: Config, credentials: ResourceCredentials):
+        self.config = config
+        self._credentials = credentials.get_credentials(self, Credentials)
```

Comment:
You are not using self._credentiails outside init - so no need for self _credentials is OK.

Reply:
Ok I will change it.